fixit/rules/no_string_type_annotation.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Set
+
+import libcst as cst
+import libcst.matchers as m
+from libcst.metadata import QualifiedNameProvider
+
+from fixit import (
+    CstContext,
+    CstLintRule,
+    InvalidTestCase as Invalid,
+    ValidTestCase as Valid,
+)
+
+
+class NoStringTypeAnnotationRule(CstLintRule):
+    """
+    Enforce the use of type identifier instead of using string type hints for simplicity and better syntax highlighting.
+    Starting in Python 3.7, `from __future__ import annotations` can postpone evaluation of type annotations
+    `PEP 563 <https://www.python.org/dev/peps/pep-0563/#forward-references>`
```

Comment:
Fix RST syntax. Note that the `_` is required for inline link.
https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets
```suggestion
    Starting in Python 3.7, ``from __future__ import annotations`` can postpone evaluation of type annotations
    `PEP 563 <https://www.python.org/dev/peps/pep-0563/#forward-references>`_
```